src/main/java/svc/managers/SMSManager.java

```diff
@@ -0,0 +1,248 @@
+package svc.managers;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.servlet.http.HttpSession;
+
+import org.joda.time.DateTime;
+import org.joda.time.Years;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.springframework.stereotype.Component;
+
+import com.twilio.twiml.Body;
+import com.twilio.twiml.Message;
+import com.twilio.twiml.MessagingResponse;
+
+import svc.dto.CitationSearchCriteria;
+import svc.managers.ViolationManager.VIOLATION_STATUS;
+import svc.models.Citation;
+import svc.models.Court;
+import svc.models.TwimlMessageRequest;
+import svc.models.Violation;
+
+@Component
+public class SMSManager {
+	@Inject
+	CitationManager citationManager;
+	@Inject
+	private CourtManager courtManager;
+	@Inject
+	private ViolationManager violationManager;
+	
+	private enum SMS_STAGE{
+		WELCOME(0),
+		READ_DOB(1),
+		READ_LICENSE(2),
+		VIEW_CITATION(3);
+		
+		private int numVal;
+		
+		SMS_STAGE(int numVal){
+			this.numVal = numVal;
+		}
+		
+		public int getNumVal(){
+			return numVal;
+		}
+	}
+	
+	
+	//a null String means validDOB, otherwise a message is returned to pass on to user
+	private String validDOB(String dob){
```

Comment:
This function would probably be better as validateDOBString or something 'verb-ish' to make reading more clear. Generally, using noun phrases as variables/classes, and verb phrases as functions will help people understand your code.